src/NUnitConsole/nunit3-console/ResultReporter.cs

```diff
@@ -118,16 +118,21 @@ public void WriteSummaryReport()
 
             WriteSummaryCount("   Tests run: ", Summary.RunCount);
             WriteSummaryCount(", Passed: ", Summary.PassCount);
-            WriteSummaryCount(", Errors: ", Summary.ErrorCount, ColorStyle.Error);
-            WriteSummaryCount(", Failures: ", Summary.FailureCount, ColorStyle.Failure);
+            WriteSummaryCount(", Failed: ", Summary.FailedCount, ColorStyle.Failure);
             WriteSummaryCount(", Inconclusive: ", Summary.InconclusiveCount);
+            WriteSummaryCount(", Skipped: ", Summary.SkipCount);
```

Comment:
Doesn't this need to be a new property, maybe TotalSkipCount which would be ignore + explicit + Skip? I am also not sure how the test suite above has 7 outputed here and 0 for the Other when they use the same variable?